app/app.go

```diff
@@ -883,7 +889,7 @@ func NewApp(
 	return app
 }
 
-func (app *ManifestApp) setAnteHandler(txConfig client.TxConfig) {
+func (app *ManifestApp) setAnteHandler(txConfig client.TxConfig, commissionRateMinMax RateMinMax) {
```

Comment:
The commission rate floor/ceil is set here, called by `NewApp()`. I can't set a new anteHandler after app creation as it is sealed. Happy to know if there's a less invasive way to do the same without modifying `NewApp()`'s signature.

Reply:
I do not think there is a way, this is a priv method anyways so does not affect anything outside